src/nbl/video/COpenGL_Queue.h

```diff
@@ -249,8 +249,7 @@ class COpenGL_Queue final : public IGPUQueue
 
     public:
         COpenGL_Queue(
-            // core::smart_refctd_ptr<IOpenGL_LogicalDevice>&& gldev,
-            IOpenGL_LogicalDevice* gldev,
+            core::smart_refctd_ptr<IOpenGL_LogicalDevice>&& gldev,
```

Comment:
no smartpointers to logical devices in queues

Reply:
resolved